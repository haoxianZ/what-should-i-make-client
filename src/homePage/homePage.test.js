import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import HomePage from './homePage'

describe(`HomePage component`, () => {

  it('renders home page', () => {
    const wrapper = shallow(<HomePage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
  
})