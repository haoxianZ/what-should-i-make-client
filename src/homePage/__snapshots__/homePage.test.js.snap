// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component renders home page 1`] = `
<div
  className="landingPage"
>
  <header>
    What should I Make?
  </header>
  <Link
    to="/about"
  >
    About
  </Link>
  <main>
    <form
      onSubmit={[Function]}
    >
      <div
        className="userbox"
      >
        <input
          id="username"
          name="username"
          placeholder="Username"
          required={true}
          type="test"
        />
        <input
          id="email"
          name="email"
          placeholder="Email"
          required={true}
          type="password"
        />
      </div>
      <div
        className="buttons"
      >
        <button
          type="submit"
        >
          Log in
        </button>
        <Link
          style={
            Object {
              "textDecoration": "none",
            }
          }
          to="/add-user"
        >
          <button>
            Sign up
          </button>
        </Link>
      </div>
    </form>
  </main>
</div>
`;
